scrapers/stackoverflow.js

```diff
@@ -0,0 +1,61 @@
+var unirest = require("unirest");
+var question = "";
+for (i = 2; i < process.argv.length; i++) {
+    question += process.argv[i] + " ";
+}
+var url = "https://api.stackexchange.com/2.2/search/advanced?key=jAAMPr3uTSgYulMJ3LY5Hw((&order=desc&sort=activity&q=" + question + "&answers=1&site=stackoverflow";
+unirest.get(url)
+    .end((response) => {
+        var response = JSON.parse(response.raw_body);
+        var QandA = new Array();
+        if (response.quota_remaining > 0) {
+            j = 0;
+            for (i = 0; i < response.items.length; i++) {
+
+                var item = response.items[i];
+                getQuestionAndAnswers(item, (data, title, link) => {
+                    var obj = new Object;
+                    obj['question'] = data.question;
+                    obj['answers'] = data.answers;
+                    obj['title'] = title;
+                    obj['link'] = link;
+                    QandA.push(obj);
+                    if (j == response.items.length - 1) {
+                        console.log(QandA);  //Final output
+                    }
+                    j++;
+                });
+
+
+            }
+        } else {
+            console.log("Request limit reached. Try after 24hrs.");
+        }
+        console.log("Requests Left: " + response.quota_remaining);
+    });
+
+
+
+function getQuestionAndAnswers(item, callback) {
+    var obj = new Object;
+    var questionId = item.question_id;
+    var urlAnswers = "https://api.stackexchange.com/2.2/questions/" + questionId + "/answers?key=jAAMPr3uTSgYulMJ3LY5Hw((&order=desc&sort=activity&site=stackoverflow&filter=withbody";
```

Comment:
Instead of hardcoding the urls sequences, could you use variables, and keep url strings at one place.